src/collection-view.js

```diff
@@ -549,14 +561,26 @@ const CollectionView = Backbone.View.extend({
   // This will also update the indices of later views in the collection in order to keep the
   // children in sync with the collection.
   addChildView(view, index) {
+    if (!(view instanceof Backbone.View)) {
```

Comment:
I'm on the fence here.  this prevents any error from occurring if you accidentally pass in something that isn't a view..  do we do this kind of bail out elsewhere?